package.json

```diff
@@ -67,7 +67,7 @@
         "classnames": "^2.2.6",
         "d2-utilizr": "^0.2.16",
         "d3-color": "^1.2.3",
-        "highcharts": "^7.1.2",
+        "highcharts": "^7.2.1",
```

Comment:
Upgraded to 7.2.1 (latest minor, to avoid 8.0.0) to fix an issue with the plot lines being drawn all the way to the center.